pkg/master/master.go

```diff
@@ -87,10 +87,10 @@ const (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	EtcdHelper    tools.EtcdHelper
-	EventTTL      time.Duration
-	MinionRegexp  string
-	KubeletClient client.KubeletClient
+	DataBaseStorage tools.StorageInterface
```

Comment:
I didn't want to have "Storage", because we already have:
"storage map[string]rest.Storage"
and having "Storage" and "storage" is really misleading in my opinion.

Changing to DatabaseStorage - I'm happy to change it in the future if you suggest better name for it.